scripts/smoketest.py

```diff
@@ -13,10 +13,22 @@
 skip_inference_tests = os.getenv("SKIP_INFERENCE_TESTS", "").lower() in {"true", "yes", "y", "1"}
 
 
-def send_document(filename, content_type, strategy="fast", output_format="application/json"):
+def send_document(
+    filename,
+    content_type,
```

Comment:
did the `content_type` input get used?

Reply:
nope, will remove it

Reply:
seems like the removal got reverted because tests weren't passing. could you either add usage (maybe pass it in to `data` in the request) or write a comment on why it needs to stay?